app/components/documents/document-card/edit-modal.js

```diff
@@ -172,5 +172,6 @@ export default class DocumentsDocumentCardEditModalComponent extends Component {
     this.isDeletingDerivedFile = false;
 
     this.args.onSave?.();
+    this.router.refresh('agenda.agendaitems.agendaitem.documents');
```

Comment:
Refreshing a specific route from a component is a bad idea, we can't make any assumptions about where this component will be used. I would look into a `reload` of the relevant models, or refresh via the `onSave` function in the specific route where the component is used.